src/mmw/apps/modeling/tasks.py

```diff
@@ -63,11 +63,11 @@ def start_histogram_job(self, json_polygon):
     the polygon so that areas can be calculated from
     the results.
     """
-    polygon = json.loads(json_polygon)
+    polygon = ensure_aoi_4d(json.loads(json_polygon))
 
     return {
         'pixel_width': aoi_resolution(polygon),
-        'sjs_job_id': histogram_start([json_polygon], self.retry)
+        'sjs_job_id': histogram_start([json.dumps(polygon)], self.retry)
```

Comment:
Note re: this: the new function to adjust the aoi will throw an exception if the RWD MultiPolygon does include more than one shape. If it's just a single shape, it'll adjust the nesting. I added a comment above this return statement to note what the validation fn will do.